docs/hazmat/primitives/symmetric-encryption.rst

```diff
@@ -340,6 +340,8 @@ Modes
     .. testcode::
 
         import os
+        
+        from cryptography.hazmat.backends import default_backend
 
```

Comment:
Very tiny nit, can you remove the newline between the two `cryptography` imports?